arrow-docs/docs/docs/datatypes/intro/README.md

```diff
@@ -0,0 +1,148 @@
+
+- [`Ior`]({{ '/docs/datatypes/ior/' | relative_url }}) - a branch in execution for three possible paths: one, two, or both
+
+- [`Const`]({{ '/docs/datatypes/const/' | relative_url }}) - tags a value with a "phantom generic" that's never instantiated, and it can be used for example to represents units or state
+
+- [`Coproduct`]({{ '/docs/datatypes/coproduct/' | relative_url }}) - constructs a new composed type from two datatypes, allowing to contain and operate on either one of them
+
+##### Error handling
+
+- [`Try`]({{ '/docs/datatypes/try/' | relative_url }}) - returns the result of executing a block of code that can fail and throw exceptions
+
+- [`Validated`]({{ '/docs/datatypes/validated/' | relative_url }}) - returns the result of aggregating multiple calculations that can fail, and it also aggregates the errors
+
+##### Reader/Writer/State
+
+- [`Kleisli`]({{ '/docs/datatypes/kleisli/' | relative_url }}) - similar to Dependency Injection and Inversion of Control, it represents a calculation with a dependency on an external context
+
+- [`Reader`]({{ '/docs/datatypes/reader/' | relative_url }}) - same as kleisli but tied to a single datatype
+
+- [`Writer`]({{ '/docs/datatypes/writert/' | relative_url }}) - represents calculations that carry over one extra aggregated value, generally a logger or reporter
+
+- [`State`]({{ '/docs/datatypes/state/' | relative_url }}) - represents a stateful calculation with a carried value that can be read from or modified, like a combination of reader and writer
+
+##### Wrappers
+
+These types wrap over some of Kotlin's collections and functions to give them capabilities related to [typeclasses]({{ '/docs/typeclasses/intro/' | relative_url }}) provided by Arrow.
+
+- [`ListKW`]({{ '/docs/datatypes/listkw/' | relative_url }})
+
+- [`SequenceKW`]({{ '/docs/datatypes/sequencekw/' | relative_url }})
+
+- [`SetKW`]({{ '/docs/datatypes/setkw/' | relative_url }})
+
+- [`MapKW`]({{ '/docs/datatypes/mapkw/' | relative_url }}) 
+
+- [`SortedMapKW`]({{ '/docs/datatypes/sortedmapkw/' | relative_url }})
+
+- [`Function0`]({{ '/docs/datatypes/function0/' | relative_url }})
+
+- [`Function1`]({{ '/docs/datatypes/function1/' | relative_url }})
+
+##### Transformers
+
+A transformer is a special kind of datatype that allows combining two datatypes to give one of them the abstractions of another
+
+- [`OptionT`]({{ '/docs/datatypes/optiont/' | relative_url }}) - gives the datatype wrapped the properties of `Option`
+
+- [`EitherT`]({{ '/docs/datatypes/eithert/' | relative_url }}) - gives the datatype wrapped the properties of `Either`
+
+- [`ReaderT`]({{ '/docs/datatypes/kleisli/' | relative_url }}) - gives the datatype wrapped the properties of `Reader`
+
+- [`WriterT`]({{ '/docs/datatypes/writert/' | relative_url }}) - gives the datatype wrapped the properties of `Writer`
+
+- [`StateT`]({{ '/docs/datatypes/statet/' | relative_url }}) - gives the datatype wrapped the properties of `State`
+
+
+##### Codata
+
+TBD
```

Comment:
Let's flag this with a TODO or issue elsewhere if we are getting rid of the `TODO` in the web

Reply:
Done